core/templates/components/forms/custom-forms-directives/thumbnail-uploader.directive.html

```diff
@@ -6,16 +6,19 @@
            ng-click="showEditThumbnailModal()"
            aria-label="upload or edit thumbnail">
       </div>
-      <i class="material-icons oppia-editor-edit-icon oppia-pencil-icon">&#xE254;</i>
-      <img ng-show="!thumbnailIsLoading"
-           ng-src="<[editableThumbnailDataUrl]>"
-           class="img-thumbnail protractor-test-custom-photo"
-           ng-style="{background: getBgColor()}">
-
-      <img ng-if="useLocalStorage && newThumbnailDataUrl"
-           ng-src="<[newThumbnailDataUrl]>"
-           ng-style="{background: localStorageBgcolor}"
-           class="img-thumbnail protractor-test-custom-photo">
+      <i class="material-icons oppia-editor-edit-icon oppia-pencil-icon">
+        &#xE254;
+      </i>
+      <oppia-svg-thumbnail-display ng-show="!thumbnailIsLoading"
+                                   [img-src]="editableThumbnailDataUrl"
+                                   [classes]="['img-thumbnail', 'protractor-test-custom-photo']"
```

Comment:
is there any specific reason for this directive to have custom classes input? (I mean is there any issue with having this class names hardcoded in the img tab itself in the component HTML file?)

Reply:
It has two types of classes that needs to be applied to img tag in `oppia-thumbnail-display`. So I had to pass it in through @Input.

Reply:
Can you check the class styles whether they are the same or can be combined? OR maybe another component using this component can set style over this component, will that work? if not, maybe define all possible classes in the component and allow a flag input for specific needs. (I'm fine either way!)